packages/connect-web/src/index.ts

```diff
@@ -156,8 +156,8 @@ export class TrezorConnectDynamicImpl implements ConnectFactoryDependencies {
         return this.getTarget().requestWebUSBDevice();
     }
 
-    public cancel() {
-        return this.getTarget().cancel();
+    public cancel(error?: string) {
```

Comment:
cc @martykan :scream:  :grin:

Reply:
Todo: would be greate to have somehow test for this

Reply:
Oops, missed that one. But it's strange TS didn't catch it, the interface should be typed correctly

Reply:
Here for example: https://github.com/trezor/trezor-suite/pull/12977/files#diff-2580f7331a6878d62ed30bb9de2c33adbf4566b5d67d8cf33f5705f48af51930R415

(this is how I found it)